h2o-algos/src/main/java/hex/ensemble/Metalearners.java

```diff
@@ -48,11 +50,16 @@ static Algorithm getActualMetalearnerAlgo(Algorithm algo) {
         return algo == Algorithm.AUTO ? Algorithm.glm : algo;
     }
 
-    static Model.Parameters createParameters(String name) {
+    static public Model.Parameters createParameters(String name) {
```

Comment:
the usual convention in java (dating from Sun Microsystems) is to use modifiers in the following order:
```
[public | protected | private] [static] [final | transient] [volatile] [abstract] …
```
Having everyone using this convention makes it easier to read.